docs/Oscilloscope.md

```diff
@@ -0,0 +1,92 @@
+# How to use Oscilloscope
+
+## Table of Content
+1. Introduction
+2. Layout
+3. Features and Components
+    - Channel Parameters
+    - Timebase and Trigger
+    - Data analysis
+    - XY Plot
+4. How to Use  
+
+## Introduction
+Pocket Science Lab contains an Oscilloscpe which has lot of functions which are available in a commercial grade Oscilloscope. The PSLab android application communicates with the Pocket Science Lab and gives the user a real experience on using a commercial Oscilloscope. This document will help a user on available functionalities, how to use Pocket Science Lab with PSLab android application and some demo videos.  
+
+## Layout
+
+The layout of the Oscilloscope graphical user interface contains three main parts. 
+1. Graph plot
+2. Function Navigation bar
+3. Functions
+
+<table>
+    <tr>
+        <td><img src="/docs/images/oscilloscope_channel_params.png"></td>
+        <td><img src="/docs/images/oscilloscope_data_analysis.png"></td>
+    </tr>
+</table>
+<table>
+    <tr>
+        <td><img src="/docs/images/oscilloscope_timebase_and_trigger.png"></td>
+        <td><img src="/docs/images/oscilloscope_xy_plot.png"></td>
+    </tr>
+</table>
+
+## Features and Components
+Available features in Pocket Science Lab regarding Oscilloscope are,
+    - 4 channels up to 2MSPS
+    - 2x Sine Wave Generators
+    - 4x PWM generators. 15nS resolution. Up to 8MHz
+
+
+Component | Name in the device
+--- | ---
+Channels | CH1, CH2, CH3, MIC
+Sine waves | SI1, SI2
+PWM Generators | SQ1, SQ2, SQ3, SQ4
+Ground | GND
+
+These features can be used using four components in the Oscilloscope. Note that GND is common for all the pins in the device.
```

Comment:
I'm not sure how we can relate `SI1` `SI2` `SQ` pins with oscilloscope as they are features in waveform generator.